### PR TITLE
Refactor ConnectionProfileDatabasePage and its subclasses

### PR DESCRIPTION
**Pull request by mlabuda**
Extract methods....

### CODE REVIEW

File: plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/ConnectionProfileDatabasePage.java [hunk 1]
```diff
@@ -10,15 +10,26 @@
...
 /**
  * An abstract wizard page for database connection profile.
  * 
  * IMPORTANT: It is assumed that the appropriate driver definition has been already created.
  * 
- * @author apodhrad
+ * @author apodhrad, mlabuda@redhat.com
  * 
  */
 public abstract class ConnectionProfileDatabasePage extends WizardPage {
```
Comment (rawagner) [inline plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/ConnectionProfileDatabasePage.java]: name & package is wrong. Should be eclipse.datatools.connectivity.ui.wizards.ExtensibleProfileDetailsWizardPage

File: plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/ConnectionProfileGenericPage.java [hunk 2]
```diff
@@ -10,125 +10,64 @@
...
-
 /**
  * A wizard page for creating new Generic JDBC (H2,..) profile.
  * 
  * 
- * @author Jiri Peterka
+ * @author Jiri Peterka, mlabuda@redhat.com
  * 
  */
 public class ConnectionProfileGenericPage extends ConnectionProfileDatabasePage {
```
Comment (rawagner) [inline plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/ConnectionProfileGenericPage.java]: name & package is wrong. Should be eclipse.datatools.connectivity.db.generic.internal.ui.GenericJDBCDBProfileDetailsWizardPage

File: plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/ConnectionProfileOraclePage.java [hunk 3]
```diff
@@ -10,27 +10,24 @@
...
-
 /**
  * A wizard page for creating new Oracle database profile.
  * 
  * 
- * @author apodhrad
+ * @author apodhrad, mlabuda@redhat.com
  * 
  */
 public class ConnectionProfileOraclePage extends ConnectionProfileDatabasePage {
```
Comment (rawagner) [inline plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/ConnectionProfileOraclePage.java]: name & package should be eclipse.datatools.enablement.oracle.internal.ui.OracleDBProfileDetailsWizardPage

File: plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/ConnectionProfileSQLServerPage.java [hunk 4]
```diff
@@ -10,136 +10,63 @@
...
-import org.jboss.reddeer.swt.impl.text.LabeledText;
-
 /**
  * A wizard page for creating new SQL Server database profile.
  * 
- * @author apodhrad
+ * @author apodhrad, mlabuda@redhat.com
  * 
  */
 public class ConnectionProfileSQLServerPage extends ConnectionProfileDatabasePage {
```
Comment (rawagner) [inline plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/ConnectionProfileSQLServerPage.java]: eclipse.datatools.enablement.msft.sqlserver.ui.connection.SQLServerDBProfileDetailsWizardPage

**Comment by odockal** [comment]
testPR

**Comment by rawagner** [comment]
GenericConnectionProfileTest is failing

**Comment by mlabuda** [comment]
Yeah, I noticed. It is because generic JDBC page does not have ports and implementation from superclass has to be hidden in this case. Now it fixed. I tried to fix as well SQL Result Test, but the problem is upstream and I did not find any workaround so far.

**Comment by rawagner** [comment]
testPR

**Pull request merged**